core/src/test/java/nu/yona/server/util/TimeUtilTest.java

```diff
@@ -19,24 +19,24 @@
 	public void getEndOfDay_anyTime_returnsTwelveOClockInGivenZone()
 	{
 		ZonedDateTime result = TimeUtil.getEndOfDay(ZoneId.of("Europe/Amsterdam"),
-				ZonedDateTime.parse("2017-08-22T20:44:11+01:00[Europe/Paris]"));
-		assertThat(result, equalTo(ZonedDateTime.parse("2017-08-23T00:00:00+01:00[Europe/Amsterdam]")));
+				ZonedDateTime.parse("2017-08-22T20:44:11+02:00[Europe/Paris]"));
```

Comment:
Given this is in the summer, the offset is 2 hours. This was always wrong but never noticed.